.pre-commit.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
```

Comment:
So what is this? Just a helper for the `changelog` action`? If so, I think it should move to `.github/helpers`

Reply:
@DevScyu